### PR TITLE
dd: align page throughout the platforms

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by leonvsc — APPROVED** [review]
Dutch pages looks good to me!

**Review by spageektti — APPROVED** [review]

File: pages.ko/linux/dd.md [hunk 1]
```diff
@@ -9,24 +9,20 @@
 
 - 4 MiB 블록이 있는 다른 드라이브에 드라이브를 복제하고, 오류를 무시하고 진행 상황을 표시:
 
-`dd if={{/dev/소스_드라이브}} of={{/dev/목적지_드라이브}} bs=4m conv=noerror status=progress`
+`dd bs=4M conv=noerror if={{/dev/소스_드라이브}} of={{/dev/목적지_드라이브}} status=progress`
 
 - 커널 랜덤 드라이버를 사용하여 랜덤 100바이트의 파일 생성:
```
[the remark this answers is not in the document]
Comment (IMHOJEONG) [reply pages.ko/linux/dd.md:14]:
```suggestion
- 커널 랜덤 드라이버를 사용하여, 특정 수의 임의 바이트 크기를 가지는 파일 생성:
```

How about this? I tried to translate it more naturally.

**Review by IMHOJEONG — APPROVED** [review]
Thank you for editing the Korean file.

File: pages.pt_BR/common/dd.md [hunk 2]
```diff
@@ -3,30 +3,22 @@
...
-`dd if={{arquivo.iso}} of=/dev/{{usb_drive}} status=progress`
+`dd if={{caminho/para/arquivo.iso}} of={{/dev/usb_drive}}`
 
-- Clona um drive para outro drive com 4 MiB block, ignora erros e mostra o progresso:
+- Clona um drive para outro drive com 4 MiB block e ignora erros:
 
-`dd if=/dev/{{drive_fonte}} of=/dev/{{drive_destino}} bs=4M conv=noerror status=progress`
+`dd bs=4194304 conv=noerror if={{/dev/drive_fonte}} of={{/dev/drive_destino}}`
 
 - Gera um arquivo com 100 bytes aleatórios utilizando o kernel random driver:
```
Comment (vitorhcl) [inline pages.pt_BR/common/dd.md:14]:
```suggestion
- Gera um arquivo com um número específico de bytes aleatórios utilizando o kernel random driver:
```

File: pages.pt_BR/osx/dd.md [hunk 3]
```diff
@@ -5,16 +5,16 @@
 
 - Cria uma unidade USB inicializável a partir de um arquivo isohybrid (tal como `archlinux-xxx.iso`):
 
-`dd if={{arquivo.iso}} of=/dev/{{unidade_usb}}`
+`dd if={{caminho/para/arquivo.iso}} of={{/dev/unidade_usb}}`
 
 - Clona uma unidade para outra unidade com bloco de 4 MB e ignora erro:
```
Comment (vitorhcl) [inline pages.pt_BR/osx/dd.md:10]:
```suggestion
 - Clona uma unidade para outra unidade com bloco de 4 MB e ignora qualquer erro:
```

File: pages.pt_BR/osx/dd.md [hunk 4]
```diff
@@ -5,16 +5,16 @@
...
 
-`dd if=/dev/{{unidade_origem}} of=/dev/{{unidade_destino}} bs=4m conv=noerror`
+`dd bs=4m conv=noerror if={{/dev/unidade_origem}} of={{/dev/unidade_destino}}`
 
 - Gera um arquivo de 100 bytes aleatórios usando o driver aleatório do kernel:
```
Comment (vitorhcl) [inline pages.pt_BR/osx/dd.md:14]:
```suggestion
- Gera um arquivo de número específico de bytes aleatórios usando o driver aleatório do kernel:
```

**Review by vitorhcl — APPROVED** [review]
Brazilian Portuguese pages LGTM after my suggestions.

File: pages.es/osx/dd.md [hunk 5]
```diff
@@ -9,24 +9,20 @@
 
 - Clona una unidad a otra unidad con un bloque de 4 MB, ignora el error y muestra el progreso:
 
-`dd if={{/dev/dispositivo_de origen}} of={{/dev/dispositivo_de destino}} bs=4m conv=noerror status=progress`
+`dd bs=4m conv=noerror if={{/dev/dispositivo_de origen}} of={{/dev/dispositivo_de destino}} status=progress`
 
 - Genera un fichero de 100 bytes aleatorios utilizando el controlador aleatorio del kernel:
```
[the remark this answers is not in the document]
Comment (kant) [reply pages.es/osx/dd.md:14]: https://github.com/tldr-pages/tldr/pull/12926

File: pages.pt_BR/osx/dd.md (hunk 4 above)
Comment (kant) [reply pages.pt_BR/osx/dd.md:14]: aleatórios --> aleatorios

File: pages.es/osx/dd.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (kant) [reply pages.es/osx/dd.md:14]: Roger that!

**Pull request merged**